lxd/events.go

```diff
@@ -129,9 +132,12 @@ func eventSend(eventType string, eventMessage interface{}) error {
 				eventsLock.Unlock()
```

Comment:
Nope, the second eventsLock happens in a go routine.

The idea is that all the go routines are spawned with the eventsLock locked then once spawned the main thread unlocks the struct.